en/app.ftl

```diff
@@ -492,6 +498,7 @@ settings-api-key-description-bolded = Only use this with apps that you trust to
 # This is a label that appears on hover to copy the API key
 settings-button-copy = Click to copy
 setting-api-key-copied = Copied!
+setting-api-key-copied-alt = Click to copy
```

Comment:
This is on the settings page, the text for screen readers:

![image](https://user-images.githubusercontent.com/4251/196904602-30a4c791-f4f2-49f6-89f3-60643a14ca33.png)